### PR TITLE
refactor: upgrade rxjs version

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

PR upgrades rxjs version. Depends on https://github.com/argoproj/argo-ui/pull/105

### CODE REVIEW

File: ui/src/app/shared/services/applications-service.ts [hunk 1]
```diff
@@ -144,15 +145,19 @@ export class ApplicationsService {
...
-            .retry()
-            .map(data => JSON.parse(data).result as models.ApplicationWatchEvent)
-            .map(watchEvent => {
-                watchEvent.application = this.parseAppFields(watchEvent.application);
-                return watchEvent;
-            });
+        return (
+            requests
+                .loadEventSource(url)
+                // .repeat()
```
Comment (rbreeze) [inline ui/src/app/shared/services/applications-service.ts:151]: Was commenting these lines purposeful?

**Review by rbreeze — APPROVED** [review]
LGTM!

File: ui/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "@types/react-router": "^4.0.27",
     "@types/react-router-dom": "^4.2.3",
     "@types/superagent": "^3.5.7",
-    "argo-ui": "git+https://github.com/argoproj/argo-ui.git",
+    "argo-ui": "git+https://github.com/argoproj/argo-ui.git#upgrade-rxjs",
```
[the remark this answers is not in the document]
Comment (alexmt) [reply ui/package.json]: Reverted.

File: ui/src/app/shared/services/applications-service.ts (hunk 1 above)
Comment (alexmt) [reply ui/src/app/shared/services/applications-service.ts]: Thank you! nice catch. Fixed

**Pull request merged**